src/ci/docker/host-x86_64/mingw-check/Dockerfile

```diff
@@ -41,6 +41,8 @@ COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/
 ENV RUN_CHECK_WITH_PARALLEL_QUERIES 1
 
 ENV SCRIPT python3 ../x.py --stage 2 test src/tools/expand-yaml-anchors && \
+           # check library crates on all tier 1 targets
```

Comment:
The list does not include x86_64-linux, I assume because other jobs are handling that? Still, should be explained in a comment.

Same for this compiler-builtins flag, why is that needed?

Reply:
I've added some comments.